### PR TITLE
[zh-cn] sync kubeadm/kubeadm-reconfigure.md

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-reconfigure.md

### CODE REVIEW

File: content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-reconfigure.md [hunk 1]
```diff
@@ -185,16 +185,23 @@ To write new manifest files in `/etc/kubernetes/manifests` you can use:
...
 ```
 
 <!--
 The `<config-file>` contents must match the updated `ClusterConfiguration`.
-The `<component-name>` value must be the name of the component.
+The `<component-name>` value must be a name of a Kubernetes control plane component (`apiserver`, `controller-manager` or `scheduler`).
 -->
 `<config-file>` 内容必须与更新后的 `ClusterConfiguration` 匹配。
-`<component-name>` 值必须是组件的名称。
+`<component-name>` 值必须是一个控制平面组件的名称（`apiserver`、`controller-manager` 或 `scheduler`）。
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-reconfigure.md:204]:
```suggestion
`<component-name>` 值必须是一个控制平面组件（`apiserver`、`controller-manager` 或 `scheduler`）的名称。
```

File: content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-reconfigure.md [hunk 2]
```diff
@@ -185,16 +185,23 @@ To write new manifest files in `/etc/kubernetes/manifests` you can use:
 -->
 要在 `/etc/kubernetes/manifests` 中编写新的清单文件，你可以使用：
 
+<!--
+# For Kubernetes control plane components
+# For local etcd
+-->
 ```shell
+# 对于 Kubernetes 控制平面组件
```
Comment (chenrui333) [inline content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-reconfigure.md:193]:
```suggestion
# Kubernetes 控制平面组件
```

File: content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-reconfigure.md [hunk 3]
```diff
@@ -185,16 +185,23 @@ To write new manifest files in `/etc/kubernetes/manifests` you can use:
...
 kubeadm init phase control-plane <component-name> --config <config-file>
+# 对于本地 etcd
```
Comment (chenrui333) [inline content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-reconfigure.md:195]:
```suggestion
# 本地 etcd
```

File: content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-reconfigure.md [hunk 4]
```diff
@@ -185,16 +185,23 @@ To write new manifest files in `/etc/kubernetes/manifests` you can use:
 -->
 要在 `/etc/kubernetes/manifests` 中编写新的清单文件，你可以使用：
```
Comment (chenrui333) [inline content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-reconfigure.md:186]:
```suggestion
要在 `/etc/kubernetes/manifests` 中编写新的清单文件，你可以使用以下命令：
```

File: content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-reconfigure.md (hunk 2 above)
Comment (chenrui333) [reply content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-reconfigure.md:193]: 不用太过于 literal 😁

**Review by chenrui333 — APPROVED** [review]
/lgtm

**Pull request merged**